src/components/Header/Header.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import propTypes from 'prop-types';
```

Comment:
I don't see prop-types listed in package.json

Reply:
'prop-types' is inbuilt react's implementation to try to address type-checking, it comes with CRA .